resources/views/admin/groups/index.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="card card--section mb-5">
+        @if ($errors->has('group_max'))
+            <div role="alert" class="alert alert-danger">
+                Group maximum size must be equal or bigger than minimum.
+            </div>
+        @endif
+        <div class="card-header">List of courses</div>
+        <table class="card-table table table-responsive">
+            <tbody>
+                @foreach ($courses as $year => $courses)
+                    <tr>
+                        <th colspan="3" class="table-active">
+                            {{ $year }} year
+                        </th>
+                    </tr>
+                    <tr>
+                        <th class="d-none d-sm-table-cell">Semester</th>
+                        <th>Course</th>
+                        <th>Min - Max</th>
+                    </tr>
+                    @foreach ($courses as $course)
+                        <tr>
+                            <td class="d-none d-sm-table-cell">{{ $course->present()->getOrdinalSemester() }}</td>
+                            @if ($course->group_max > 0)
+                                <td><a href="/admin/{{ $course->id }}/groups" /a>{{ $course->name }}</td>
+                            @else
+                                <td>{{ $course->name }}</td>
+                            @endif
+                            <td>
```

Comment:
This should be in a view edit, not in the index. I saw it in the view and was a little confusing.

Reply:
I don't think it would be better to have to open a page just edit the groups size of a single course

Reply:
It was just a suggestion, but I will let @fntneves decide on that.